### ISSUE TITLE
Cant get ahold of you any other way , so  i might as well try it over issues, need help for sublime text and your guna theme

### ISSUE THREAD

**Issue by washingleavesandthensome23-434** [poucotm/Guna#19]
problem i have is , ive did some custom colors configuration and then entered manual , 

ive entered into the console this command

Guna : Tweak Theme

dont work 

btw do you have an update to it , would love an update

**Comment by poucotm** [comment]
The command should be typed in the command palette (ctrl+shift+p) NOT the console.

![image](https://user-images.githubusercontent.com/19210389/79626447-393cef80-816b-11ea-9b69-243b6bdbd22f.png)

**Comment by washingleavesandthensome23-434** [comment]
oh , lols, gonna try that and get back atcha, thanks in advance

**Comment by washingleavesandthensome23-434** [comment]
ok works, btw how in hell do i get your layout , it sure as fuck dont look like that over here , ive downloaded per pakagemanager your guna theme , yet still i get a pretty basic layout , looks like this> instead like yours, i do use a fontsize of 26 since i use a 4k display

https://www48.zippyshare.com/v/4wBSlR15/file.html

**Comment by poucotm** [comment]
Try the following settings :

	"tweak_mode": "manual",
	"base-color": "#0a0a0a",
	"sidebar_font.color": "#04a328",
	"sidebar_font.color.selected": "#04a328",

![image](https://user-images.githubusercontent.com/19210389/80103692-49f8c580-85b0-11ea-8dcc-0cd3114794d5.png)

And I'm sorry, I've never seen my theme in the High DPI monitor. I want but don't have chance ...

**Comment by washingleavesandthensome23-434** [comment]
thats a shame , could you alternatively upload your sublime text 3 folder , maybe its just something with my install thats missing something compared to yours, btw it shouldnt be difficult to test on a 4k display , you can get them cheap , you have no way of getting ahold of a 4k display to test it out on ? , anyhow would be appreciated if you could help out with getting your theme to look like it should over at my side , would be really appreciated alot , since as you can see it looks like default sublime text 3 but with your theme mixed in instead of the custom layout of yours

**Comment by washingleavesandthensome23-434** [comment]
please update your theme for ST4 ...i beg of you

**Comment by poucotm** [comment]
Based on [jfcherng](https://github.com/jfcherng)'s request, I added `sidebar_button_control` for ST4. But I'm not sure it will work with ST4.  Maybe updating the plug-in for ST4 will take a long time, if possible, it will be done after ST4 publicly released.

**Comment by washingleavesandthensome23-434** [comment]
i see ...well great news indeed , keep it up my dude, your theme is THE Best for Sublime bar none just letting you know

**Comment by washingleavesandthensome23-434** [comment]
btw one more issue ive got , is there a way to double the size of all icons aka buttons such as the sidebar/console button , actually i want ALL "buttons" to be double size ..except the tabs close one
find history button , regular expression , case sensitive, whole word, wrap, in matches and highlight
not just sidebar, would be very nice and would appreciate the help

**Comment by poucotm** [comment]
I added @2x button icons more for High DPI (Guna v1.7.10).
Could you show what you're looking in High DPI monitor ?

**Comment by washingleavesandthensome23-434** [comment]
i cant find the double sized icons option in Guna.sublime-settings

**Comment by poucotm** [comment]
As I know, sublime selects @2x named icon image under High DPI mode and automatically changes the margin for it. But I never checked it.
I think that you can manually change as the following for a test (highlight button).

in Packages\Guna\themes\Guna.sublime-theme
from
	{
		"class": "icon_highlight",
		"layer0.texture": "Guna/assets/simple/widget/button/highlight.png",
		"layer0.tint": [71, 86, 102],
		"layer0.opacity": 1,
		"content_margin": [10, 10]
	},
to
	{
		"class": "icon_highlight",
		"layer0.texture": "Guna/assets/simple/widget/button/highlight.png",
		"layer0.tint": [71, 86, 102],
		"layer0.opacity": 1,
		"content_margin": [20, 20]
	},

**Comment by washingleavesandthensome23-434** [comment]
ok that works , hope you make this an optional feature or such for all mentioned buttons

**Comment by poucotm** [comment]
I've added the scale setting (Guna v1.9.0).
You can change it with other settings in Preferences > Package Settings > Guna > Dark (Light) theme settings. However the widgets (clock/weather) are not ready yet.

![image](https://user-images.githubusercontent.com/19210389/82135936-26d5d480-9843-11ea-844e-0d90a8ea5521.png)

**Comment by washingleavesandthensome23-434** [comment]
woohhhh..nice ....cant wait

**Comment by washingleavesandthensome23-434** [comment]
ok so as said yeah, weather/clock and the sidebar icon need a size fix and readd manual mode , since without it the base color hexcode i set in guna settings wont apply globally properly

**Comment by poucotm** [comment]
The widget scale option - 1 or 1.33 is added (v1.9.5) in Preferences > Package Settings > Guna > Dark (Light) theme settings.

**Comment by washingleavesandthensome23-434** [comment]
THANKS A TON!!! you are the best! needed this my dude , i indeed appreciate you updating your mod for high dpi support , thanks again , guna nr.1 Theme!

p.s only thing missing is the sidebar icon , its still small

**Comment by poucotm** [comment]
If you're using A file Icon, it overrides theme's settings. You can change the file type icon size in **Preferences > Package Settings > A File Icon**  like `  "size": 12`.
BTW, there are 3-type icons @1x, @2x, @3x (\Sublime Text 3\Packages\zzz A File Icon zzz\patches). But you don't want to use DPI scaling as you said. Simply, replace all @1x images as @3x to see not blurred icons.

@1x image
![image](https://user-images.githubusercontent.com/19210389/83960775-916cb400-a8c7-11ea-8381-705cae52ba90.png)
@3x image
![image](https://user-images.githubusercontent.com/19210389/83960814-0b9d3880-a8c8-11ea-808f-48c744124ce0.png)

**Comment by poucotm** [comment]
I've added the function that controls the size setting of 'A File Icon' automatically (Guna v1.9.8). Yon don't need to change manually. You can also change file type icons of 'A File Icon' to use High DPI (@3x) images by running **Preferences > Guna > Upscale Icon**.

**Comment by washingleavesandthensome23-434** [comment]
well this is additionally cool beans for sure , thats once full Open Files Sidebar supports filetype icons , since it currently doesnt but a issue ticket for its inclusion exists , but my issue is that the   Sidebar Icon to open or close the sidebar at the bottom left for Open Files is still tiny aka this still needs implementing

**Comment by washingleavesandthensome23-434** [comment]
any news on my request pertaining the open files sidebar icon size, hasnt been updated yet?

**Comment by washingleavesandthensome23-434** [comment]
> I've added the function that controls the size setting of 'A File Icon' automatically (Guna v1.9.8). Yon don't need to change manually. You can also change file type icons of 'A File Icon' to use High DPI (@3x) images by running **Preferences > Guna > Upscale Icon**.

as said still needs the open files sidebar icon size updated , its still tiny , thanks alot my man

**Comment by poucotm** [comment]
The size of sidebar icon can be set with `"scale":xx, "sidebar_font.size": xx`
Do you mean it doesn't work ?

![image](https://user-images.githubusercontent.com/19210389/89638962-b7263180-d8e7-11ea-9b21-ffe5afaaccba.png)

**Comment by washingleavesandthensome23-434** [comment]
this doesnt change the actual sidebar icon of the open files sidebar theres 2 different sidebars as you may know one is the current projects sidebar and the other the open files sidebar as you have seen in my screenshots

your screenshot shows you using the current projects sidebar NOT the open files sidebar, try it with current files sidebar and youll see that icon is small

**Comment by poucotm** [comment]
https://www110.zippyshare.com/v/nZXmjTM6/file.html link is broken. plz, just copy/paste your screenshot here. The above picture shows open files folder by running File > Open Folder.

**Comment by washingleavesandthensome23-434** [comment]
no i meant the actual sidebar activate icon at the bottom left as demonstrated by yourself here >


https://user-images.githubusercontent.com/19210389/82135936-26d5d480-9843-11ea-844e-0d90a8ea5521.png

**Comment by poucotm** [comment]
You mean this ? (captured with scale=4)

![image](https://user-images.githubusercontent.com/19210389/89911046-f0cea380-dc2b-11ea-8bab-534b59fbb335.png)

**Comment by washingleavesandthensome23-434** [comment]
yep and it dont scale no matter the settings of scale thats the issue

**Comment by poucotm** [comment]
As I remember, you're using Sublime Text 4. I applied scaling on 'sidebar_button_control' which is used only for Sublime 4.
I hope it would work for you. Please, check with v2.1.2 version.

**Comment by washingleavesandthensome23-434** [comment]
yep now it works ...BUT there  a size issue , the sidebar icon size increases with the scale setting but its like half the size of the other icons , once fixed itll be perfect

**Comment by poucotm** [comment]
@washingleavesandthensome23-434 , I added `switch_icon_scale` parameter for this update (v2.5.0). You can change the sidebar icon  scale with it.
